xpallets/assets/src/weights.rs

```diff
@@ -40,22 +40,22 @@ pub trait WeightInfo {
 pub struct SubstrateWeight<T>(PhantomData<T>);
 impl<T: frame_system::Trait> WeightInfo for SubstrateWeight<T> {
     fn transfer() -> Weight {
-        (275_666_000 as Weight)
+        (254_607_000 as Weight)
             .saturating_add(T::DbWeight::get().reads(8 as Weight))
             .saturating_add(T::DbWeight::get().writes(6 as Weight))
     }
     fn force_transfer() -> Weight {
-        (288_613_000 as Weight)
+        (260_192_000 as Weight)
             .saturating_add(T::DbWeight::get().reads(8 as Weight))
             .saturating_add(T::DbWeight::get().writes(6 as Weight))
     }
     fn set_balance(_n: u32) -> Weight {
```

Comment:
@liuchengxu Do we need the parameter `n`?

Reply:
No, it's unused.